source/clients/admin/microkernel_architecture/microkernel_modules/functionalities_manager/FunctionalitiesManager.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef FUNCTIONALITIES_MANAGER_HPP_
+# define FUNCTIONALITIES_MANAGER_HPP_
+
+# include "AFunctionalitiesManager.hpp"
+
+class FunctionalitiesManager : public AFunctionalitiesManager
+{
+public:
+    FunctionalitiesManager(QObject *parent = 0);
```

Comment:
From Qt documentation `QObject::QObject(QObject * parent = 0)`
http://doc.qt.io/qt-5/qobject.html#QObject

So why setting it to `0` instead of `NULL` ? Honestly I don't know

Otherwise, `NULL` seems to be deprecated in c++ 11
It should be replaced by `nullptr`
http://stackoverflow.com/questions/9299101
1st answer, 13th bullet
But I does not explain why `0`

Litteraly, `NULL` is just a `#define NULL 0`, so it's basically the same
So maybe we should use `nullptr` ?